deploy/80_rh-ssh.Namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "${{SSHD_NAMESPACE}}"
```

Comment:
Where does SSHD_NAMESPACE come from? What is the value expected to be?

Reply:
I don't think the double curly brace "{{" are needed here.  I think you want single curly brace.

We use double curly brace for the ALLOWED_CIDR_BLOCKS var because this is a list, and the double curly brace expands this into an actual list.  The single curly brace should expand into a string.

Reply:
We don't _need_ the variable, but the Jenkins jobs @luis-falcon is setting up to generate the SSH key configmaps will need to know what namespace to deposit them.  So my thought was just to keep that name, if not in one place, then at least in one repo (app-interface).

Ack on the single curly brace.  ¯\_(ツ)_/¯

Reply:
By the way, what software is performing the variable substitution here? In other words, where are these curly brace rules coming from? Are they meant to be Jinja templates or something else?